example/src/simple/component/master.js

```diff
@@ -0,0 +1,140 @@
+/**
+ *
+ * @module component/master
+ * @author Greg Babula [greg.babula@mlb.com]
+ * @desc entry point for all component specific functionality
+ * jQuery and Bootstrap provided for example, however neither are required
```

Comment:
Something like this: https://github.com/MajorLeagueBaseball/g5-component/pull/20/commits/bb5c81ba428444e06d3da43183fa4bd78f3d4d95